azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/invoker/storage/EventProcessor.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.function.invoker.storage;
```

Comment:
You can create a new package `com.microsoft.azure.maven.function.invoker.eventhub` and put this file into it.